base/xx-info

```diff
@@ -338,10 +338,8 @@ case "$1" in
     echo "XX_ARCH=${TARGETARCH}"
     echo "XX_MARCH=${XX_MARCH}"
     echo "XX_VENDOR=${XX_VENDOR}"
-    if [ "$TARGETOS" = "linux" ]; then
-      echo "XX_PKG_ARCH=${XX_PKG_ARCH}"
-      echo "XX_TRIPLE=${XX_TRIPLE}"
-    fi
+    echo "XX_PKG_ARCH=${XX_PKG_ARCH}"
```

Comment:
PKG_ARCH should probably remain under condition as we only support linux package managers.

Reply:
Yes indeed we don't support Cygwin pkgs